src/App.js

```diff
@@ -14,6 +14,13 @@ function App() {
 
 	return (
 		<div className='App'>
+			 <div>
+      			<input
+       			 type="text"
+       			 name="token"
+        		 placeholder="enter token"/>
+				 <input type="submit"/>
+             </div>
 			<Link to='/FirebaseTesting'>FirebaseTesting</Link>
```

Comment:
@AshlingH if you wrap the `input` in a `form`, you'll have some built-in submit functionality.